### PR TITLE
fixed zig.json bug

### PR DESCRIPTION
**Pull request by max2000warlord**

### CODE REVIEW

File: snippets/zig.json [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
-				".optimize = optimize,",
-				"});",
-				"\n\n\n b.installArtifact(exe);",
-		],
-		"description": "Building an exe"
-	}
+  "Import": {
+    "prefix": "import",
+    "body": ["const ${1} = @import(\"${1}\")"],
+    "description": "Importing Librarys"
```
Comment (fbrunel) [inline snippets/zig.json:5]: @max2000warlord there is a typo here, should be spelled "libraries"

Comment (max2000warlord) [reply snippets/zig.json]: Didn't notice that lol. Looks like it's in the original code, too. Fixed.

**Comment by Eveeifyeve** [comment]
Superseeded by #531.

**Comment by OkelleyDevelopment** [comment]
Similar to the other PR comments, https://github.com/rafamadriz/friendly-snippets/pull/531 is merged so I'm declining this PR. Update your plugins and it should resume working as normal. Thanks!

**Pull request closed without merging**